packages/itwin/property-grid/scripts/start-test-viewer.js

```diff
@@ -1,9 +1,18 @@
 /*---------------------------------------------------------------------------------------------
- * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
- * See LICENSE.md in the project root for license terms and full copyright notice.
- *--------------------------------------------------------------------------------------------*/
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
 const crossEnv = require("cross-env");
-const path = require('path').resolve(__dirname, '../../../../apps/test-viewer')
+const path = require("path").resolve(__dirname, "../../../../apps/test-viewer");
+const watch = process.argv[2] && process.argv[2] === "--watch";
+let options = {
+  shell: true,
+};
+let startDevPrefix = ":no-watch";
+if (watch) {
+  options = {};
+  startDevPrefix = "";
+}
```

Comment:
I think it would be clearer if we just had this:
```
const scriptName = watch ? "start:dev" : "start:dev:no-watch";
const options = watch ? undefined : { shell: true };
```

Reply:
Changed

Reply:
Renamed